.changeset/orange-bees-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pantheon-systems/next-wordpress-starter": patch
```

Comment:
I think this should be a minor version bump, not positive.

Reply:
After reading up on this I agree, since this is feature work it should be considered minor.